Shuttle/AppDelegate.m

```diff
@@ -24,15 +24,9 @@ - (void) awakeFromNib {
     regularIcon = [NSImage imageNamed:@"StatusIcon"];
     altIcon = [NSImage imageNamed:@"StatusIconAlt"];
     
-    // Check for AppKit Version, add support for darkmode if > 10.9
-    BOOL oldAppKitVersion = (floor(NSAppKitVersionNumber) <= NSAppKitVersionNumber10_9);
-    
-    if (!oldAppKitVersion)
-    {
-        // 10.10 or higher, add support to icon for auto detection of Regular/Dark mode
-        [regularIcon setTemplate:YES];
-        [altIcon setTemplate:YES];
-    }
```

Comment:
@vmdanilov Could you please explain this change? I thought the conditional was necessary for backwards compatibility with 10.9?